roles/frr/templates/frr_loadbalancer.conf.j2

```diff
@@ -36,27 +36,63 @@ router bgp {{ frr_local_as }}
 exit
 !
 ip prefix-list internal seq 5 permit {{ frr_internal_v4 }} ge 32 le 32
+{% for item in frr_internalbackup_v4 %}
```

Comment:
frr_internalbackup_v4 has to be added to defaults.yml

Reply:
I think it doesn't, because the default service type is leaf

Reply:
I think it is best practice if used variables are initialized with a value that the template is usable.